p2p/peermanager.go

```diff
@@ -149,6 +149,10 @@ type PeerManagerOptions struct {
 	// for testing. A score of 0 is ignored.
 	PeerScores map[NodeID]PeerScore
 
+	// PrivatePeerIDs defines a set of NodeID objects which the PEX reactor will
+	// consider private and never gossip.
+	PrivatePeerIDs map[NodeID]struct{}
```

Comment:
nit: make it consistent with `PersistentPeers`
```suggestion
	PrivatePeers map[NodeID]struct{}
```